mysql_schema_configuration/mysql_schema_configuration.tf

```diff
@@ -32,6 +32,8 @@ resource "aws_ssm_parameter" "username" {
   value = "${mysql_user.user.user}"
 
   key_id = "${lookup(var.optional_parameters, "username_kms_key_id", "")}"
+
+  tags = "${var.optional_ssm_parameter_tags}"
```

Comment:
No reason to think that the tags could vary for the username and the password ?

Reply:
they could obvisouly, but I think it's okay to only support both for the moment. What do you think?

Reply:
Make sense, let see with the usage what happen :+1: